amr-wind/helics.H

```diff
@@ -0,0 +1,60 @@
+#ifndef HELICS_H
+#define HELICS_H
+
+#include "AMReX_AmrCore.H"
+#include "amr-wind/core/SimTime.H"
+#include "amr-wind/CFDSim.H"
+
+#ifdef AMR_WIND_USE_HELICS
+#include "helics/cpp98/CombinationFederate.hpp"
+#include "helics/cpp98/helics.hpp"
+#include "helics/cpp98/Federate.hpp"
+#endif
+
+namespace amr_wind {
+
+class helics_storage
+{
+public:
+    explicit helics_storage(CFDSim& sim);
+    ~helics_storage();
+
+    void pre_advance_work() const;
+    void send_messages_to_controller();
+    void recv_messages_from_controller();
+    bool is_activated() const { return helics_activated; }
+
+#ifdef AMR_WIND_USE_HELICS
+    HelicsTime currenttime = 0.0;
+    std::string fedinitstring = "--federates=1";
+#endif
+    double deltat =
+        0.01; // what is this for? should we use m_time.deltaT() instead?
+
+    int m_num_turbines{1};
+
+    double m_inflow_wind_speed_to_amrwind{8.0};
+    double m_inflow_wind_direction_to_amrwind{270.0};
+
+    std::vector<double> m_turbine_power_to_controller;
```

Comment:
I feel like we should just use `amrex::Vector` and avoid `std` to be consistent.